docs/src/dev-docs/changelog.rst

```diff
@@ -24,6 +24,13 @@ changelog <https://keepachangelog.com/en/1.1.0/>`_ format. This project follows
 Unreleased
 ----------
 
+Added
+#####
+
+- Added a refactored composition model ``MetatranCompositionModel`` to be be phased in to
+  individual architectures. This can be used as a standalone model and will be extended
+  to support more complex targets in the future.
+
```

Comment:
Since this is an internal change, it does not need to be added to the user-facing changelog

```suggestion
```

Reply:
noted